x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/action/XPackUsageResponseTests.java

```diff
@@ -68,15 +71,15 @@ public NewUsage(final StreamInput in) throws IOException {
 
         @Override
         public TransportVersion getMinimalSupportedVersion() {
-            return newVersion.transportVersion;
+            return newVersion;
         }
 
     }
 
     public void testVersionDependentSerializationWriteToOldStream() throws IOException {
         final XPackUsageResponse before = new XPackUsageResponse(List.of(new OldUsage(), new NewUsage()));
         final BytesStreamOutput oldStream = new BytesStreamOutput();
-        oldStream.setVersion(VersionUtils.randomVersionBetween(random(), oldVersion, VersionUtils.getPreviousVersion(newVersion)));
```

Comment:
Moving the random versions to the setup, rather than having two sets of random selections